macros/src/actix.rs

```diff
@@ -140,7 +116,7 @@ pub fn emit_v2_operation(attrs: TokenStream, input: TokenStream) -> TokenStream
     } else if is_impl_trait {
         quote!((|| #block)())
     } else {
-        quote!((|| async #block)())
+        quote!(async move #block)
```

Comment:
Can we make this a move closure? `move || #block` and `move || async #block`?

Reply:
Applied. Out of curiosity, what's the difference of using closure in this case?

Reply:
We're storing the result in a wrapper (below). If we only have a block, then we can't support handlers which have early returns.